substrate/client/rpc-servers/Cargo.toml

```diff
@@ -30,7 +31,6 @@ tokio = { features = ["parking_lot"], workspace = true, default-features = true
 tower = { workspace = true, features = ["util"] }
 tower-http = { workspace = true, features = ["cors"] }
 
-# Dependencies outside of the polkadot-sdk workspace
-# which requires hyper v1 and http v1
-http = "1.1"
+# Dependencies outside the polkadot-sdk workspace
+# which requires hyper v1
 hyper = "1.3"
```

Comment:
Wait, why we still have this here? @ggwpez can we not just put this as `hyperv1` into the workspace Cargo.toml?

Reply:
It introduced from https://github.com/paritytech/polkadot-sdk/pull/4730/files#diff-0197cf029495b0a78f7c94cf0c846c99edc5058479835adc815e4e7cd59ba392R33-R36

Reply:
What happens if you remove it?

Reply:
It doesn't work with prometheus and some other crates

I'll fix it a follow PR I'm prepping